src/state_machine.rs

```diff
@@ -1704,9 +1740,8 @@ mod test {
     }
 
     fn get_rng() -> impl RngCore + CryptoRng {
-        let rnd = OsRng;
         //rand::rngs::StdRng::seed_from_u64(rnd.next_u64()) // todo: fix trait `rand_core::RngCore` is not implemented for `StdRng`
-        rnd
+        OsRng
```

Comment:
Do you want to just get rid of this function?  We are embedding `OsRng` into various `SigningRound` functions instead of doing the nastiness to pass it in, so even if we did this it wouldn't actually solve the problem this function wants to solve.

Reply:
Will do :)

Reply:
Thanks!  

We could also have changed `SigningRound` to always pass in the `<RngCore+CryptoRng>` arg that we do elsewhere, and we probably should do that eventually, but it's not necessary for this PR XD